app/stac_api/admin.py

```diff
@@ -44,8 +45,21 @@ class CollectionLinkInline(admin.TabularInline):
 
 
 @admin.register(Collection)
-class CollectionAdmin(admin.ModelAdmin):
+class CollectionAdmin(admin.GeoModelAdmin):
+    modifiable = False  # to prohibit editing extent_geometry
+    readonly_fields = ['cache_start_datetime',
+                       'cache_end_datetime',
+                       'summaries'
+                       ]
     inlines = [ProviderInline, CollectionLinkInline]
+    # customisation of the geometry field
+    # https://docs.djangoproject.com/en/1.10/_modules/django/contrib/gis/admin/options/
+    map_template = finders.find('admin/ol_swisstopo.html')  # custom swisstopo
+    if os.environ['APP_ENV'] == 'local':
```

Comment:
`APP_ENV` is part of the django settings and should be read from it and not directly from os.environ